client/services/tutors.js

```diff
@@ -1,2 +1,66 @@
-import axios from "axios"
+import api from "./apiConfig"
+
+export const getTutors = async () => {
+  const response = await api.get("/tutors")
+  const tutors = response.data;
+  return tutors;
+};
+
+export const getTutor = async (id) => {
+  const response = await api.get(`/tutors/${id}`)
+  const tutor = response.data;
+  return tutor;
+
+};
+
+export const updateTutor = async (id, data) => {
+  const response = await api.put(`/tutors/${id}`, data)
+  const updatedTutor = response.data;
```

Comment:
not 100% but in react architecture repo for update has string interpolation on 36 of apiUrl
https://git.generalassemb.ly/sei-nyc-ducks/react-architecture/blob/master/src/services/products.js

Reply:
Not sure either since i'm under the impression that we are importing the api from apiConfig will touch base with Instructors.

Reply:
reconciled